static/src/inline-svgs/icon/hand.svg

```diff
@@ -0,0 +1,10 @@
+<?xml version="1.0" encoding="utf-8"?>

```

Comment:
Need to run this through https://jakearchibald.github.io/svgomg/ (not sure why this isn't automatic :/ )